### ISSUE TITLE
unsupported two device

### ISSUE THREAD

**Issue by Ramulet** [rytilahti/python-miio#1890]
Use `miiocli device --ip <ip address> --token <token> info`.
# first
Model: lumi.gateway.v3
Hardware version: MW300
Firmware version: 1.4.1_176

# second
Model: cuco.plug.cp1
Hardware version: esp8285
Firmware version: 2.0.1_0011

**Comment by rytilahti** [comment]
You need to be more descriptive about what is not working. At least the second one should be working fine using the genericmiot (available on the git master branch) after #1898 is merged:
```
miiocli genericmiot --ip 127.0.0.1 --token 00000000000000000000000000000000 status
Running command status
Service Switch (switch)
        Switch Status (switch:on, access: RW): True (<class 'bool'>, )
Service Indicator Light (indicator-light)
        Switch Status (indicator-light:on, access: RW): False (<class 'bool'>, )
```

**Comment by Ramulet** [comment]
> You need to be more descriptive about what is not working. At least the second one should be working fine using the genericmiot (available on the git master branch) after #1898 is merged:
> 
> ```
> miiocli genericmiot --ip 127.0.0.1 --token 00000000000000000000000000000000 status
> Running command status
> Service Switch (switch)
>         Switch Status (switch:on, access: RW): True (<class 'bool'>, )
> Service Indicator Light (indicator-light)
>         Switch Status (indicator-light:on, access: RW): False (<class 'bool'>, )
> ```

sorry,i had cloned a wrong branch.
